documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -5,127 +5,255 @@
+----
+<1> Kafka Connect cluster group that the instance belongs to.
+<2> Kafka topic that stores connector offsets.
+<3> Kafka topic that stores connector and task status configurations.
+<4> Kafka topic that stores connector and task status updates.
+<5> Converter to transform message keys into JSON format for storage in Kafka.
+<6> Converter to transform message values into JSON format for storage in Kafka.
+<7> Schema enabled for converting message keys into structured JSON format.
+<8> Schema enabled for converting message values into structured JSON format.
+<9> Replication factor for the Kafka topic that stores connector offsets.
+<10> Replication factor for the Kafka topic that stores connector and task status configurations.
+<11> Replication factor for the Kafka topic that stores connector and task status updates.
+
+NOTE: In this example, JSON converters are specified. The converters you specify must be in a data format supported by Kafka.
+
+A distributed Kafka Connect cluster has a group ID and a set of internal connector topics.
 Kafka Connect instances are configured by default with the same:
 
 * Group ID for the Kafka Connect cluster
 * Kafka topic to store the connector offsets
 * Kafka topic to store connector and task status configurations
 * Kafka topic to store connector and task status updates
 
-If multiple different Kafka Connect instances are used, these settings must reflect each instance.
+If multiple different Kafka Connect instances are used, these settings must be unique for the workers of each Kafka Connect cluster created.
 
-[discrete]
-== Example YAML showing Kafka Connect configuration
+.Worker configuration that must be unique for each Kafka Connect instance
 [source,yaml,subs="attributes+"]
 ----
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect
-metadata:
-  name: my-connect
+# ...
 spec:
-  # ...
   config:
     group.id: my-connect-cluster
     offset.storage.topic: my-connect-cluster-offsets
     config.storage.topic: my-connect-cluster-configs
     status.storage.topic: my-connect-cluster-status
-  # ...
 ----
 
-[discrete]
-== Connectors
+The group ID identifies the Kafka Connect worker nodes. Kafka Connect clusters cannot share the ID or topics as it will create errors.
+The names of the connectors used by each Kafka Connect instance must also be unique.
 
-Connectors are configured separately from Kafka Connect.
-The configuration describes the source input data and target output data to feed into and out of Kafka Connect.
-The external source data must reference specific topics that will store the messages.
+You can also deploy connector plugins using the `build` configuration of the `KafkaConnect` resource.
+Connector plugins provide specific configuration for the creation of connector instances.
 
-Kafka provides two built-in connectors:
+After connector plugins have been deployed, you can use Strimzi’s `KafkaConnector` custom resource or the Kafka Connect API to manage connector instances.
```

Comment:
This might be better. I would probably not say that the plugins are deployed. Only the connector instances can be deployed.

```suggestion
After connector plugins have been added to the Kafka Connect deployment, you can use Strimzi’s `KafkaConnector` custom resource or the Kafka Connect API to manage connector instances.
```